### PR TITLE
Add Pearson icon

### LINKED ISSUE #5865: Pearson
**Brand Name:** Pearson
**Website:** https://www.pearson.com/
**Alexa rank:**
  

**Official resources for icon and color:**

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
pearson
**Issue:** closes #5865 

**Similarweb rank:** 1,504. See https://www.similarweb.com/website/pearson.com

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from website https://www.pearson.com
Color: Taken from logo `#000000`. Also mentioned in the [branding guidelines](https://www.pearson.com/content/dam/corporate/global/pearson-dot-com/files/logos/Pearson_Guidelines_Logo.pdf) which I cannot find a landing page for.

### CODE REVIEW

**Comment by adamrusted** [comment]
Also worth noting the exact same logo is used for Pearson VUE, the online testing platform. We might want to add this in as an AKA alias.

**Comment by PratyushJayachandran** [comment]
> Also worth noting the exact same logo is used for Pearson VUE, the online testing platform. We might want to add this in as an AKA alias.

Added and pushed commit. Thanks for the suggestion 💯

**Comment by LitoMore** [comment]
image


@PratyushJayachandran I picked the resource from their homepage, but I found some difference

**Comment by PratyushJayachandran** [comment]
> image
> 
> @PratyushJayachandran I picked the resource from their homepage, but I found some differences.

I have tried to fix it. Hope the new commit resolves it. Thanks for the review.

**Review by LitoMore — CHANGES_REQUESTED** [review]
image

Hey @PratyushJayachandran, please review your changes before committing. Thank you.

**Comment by PratyushJayachandran** [comment]
> image
> 
> Hey @PratyushJayachandran, please review your changes before committing. Thank you.

Oops my bad. It was looking fine in my editing app but then after optimization it becomes this. Forgot to recheck this time.

I'll fix it.

**Comment by PratyushJayachandran** [comment]
> Hey @PratyushJayachandran, please review your changes before committing. Thank you.

I fixed it.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -8846,6 +8846,16 @@
...
             "source": "https://www.peakdesign.com"
         },
+        {
+            "title": "Pearson",
+            "hex": "000000",
+            "source": "https://www.pearson.com",
+            "aliases": {
+                "aka": [
+                    "Pearson VUE"
+                ]
```
Comment (adamrusted) [inline _data/simple-icons.json:8856]: I know I suggested the `aka` of Pearson VUE, but this is probably a better way to handle it, given their hex codes are different...

```suggestion
                "dup": [
                    {
                        "title": "Pearson VUE",
                        "hex": "008FB4"
                    }
                ]
```

**Review by adamrusted — APPROVED** [review]
Icon itself is a perfect match, just an amendment to the JSON, which I should've proposed instead of the 'aka' entry and then we're good to go. I'll merge this in myself - save you doing it and re-requesting a review @PratyushJayachandran 🎉

**Pull request merged**